metrics/collectors/images/images_collector.py

```diff
@@ -113,6 +114,7 @@ def collect(self):
                             "release": series,
                             "current_or_pending": current_or_pending,
                             "architecture": arch,
+                            "preinstalled": preinstalled,
```

Comment:
I asked the release team, and apparently "preinstalled" is best referred to as an image "type". So I think we should add a "type" tag and set it to "live", "preinstalled", and similar. (Can we easily identify an image type for all the images?)

Reply:
Amended, please re-review